packages/lambda-powertools-logger/index.d.ts

```diff
@@ -0,0 +1,51 @@
+import CorrelationIds from "@dazn/lambda-powertools-correlation-ids";
+export = dazn__lambda_powertools_logger;
+declare enum LogLevels {
+  DEBUG = 20,
+  INFO = 30,
+  WARN = 40,
+  ERROR = 50,
+}
+
+declare class dazn__lambda_powertools_logger {
+  correlationIds: CorrelationIds;
+  originalLevel: string;
+  level: string;
+  constructor(argument?: { correlationIds?: CorrelationIds; level?: string });
+
+  appendError<T>(params: T, err: Error): T & Error;
+
+  debug(message: string, params?: any): void;
```

Comment:
For the `params` arg here and others below, instead of using any, it might be better to use the Record utility type. See usage below:
```suggestion
  debug(message: string, params?: Record<string, any>): void;
```